server/src/schema/tables/user.table.ts

```diff
@@ -37,6 +37,9 @@ export class UserTable {
   @Column({ default: '' })
   password!: Generated<string>;
 
+  @Column({ nullable: true })
+  pincode!: Generated<string> | null;
```

Comment:
Should we have it not null and default to an empty string?

Reply:
A user does not need to have a pin set, right? If that's the fact I'd strongly vote for null, since that's the intended value for something that's not set.